source/MRViewer/MRViewer.h

```diff
@@ -532,6 +532,9 @@ class MRVIEWER_CLASS Viewer
     // Window focus signal
     using PostFocusSignal = boost::signals2::signal<void( bool )>;
     PostFocusSignal postFocusSignal;
+    // Plugin enabled/disabled signal
+    using PluginEnabledChangedSignal = boost::signals2::signal<void( StateBasePlugin *, bool )>;
+    PluginEnabledChangedSignal pluginEnabledChangedSignal;
```

Comment:
can we move logic in ImGuiMenu? or ribbon Menu? Viewer should not be aware of plugins